From cc732a715ad36f4dddb35355e2f3ff3ab5715cda Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maciej=20Kr=C3=BCger?= <mkg20001@gmail.com>
Date: Sun, 28 Feb 2021 05:58:09 +0100
Subject: [PATCH] libdir patch

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index a39e3d734..e482e93b3 100644
--- a/meson.build
+++ b/meson.build
@@ -16,7 +16,7 @@ desktopdir = join_paths(datadir, 'applications')
 x_sessiondir = join_paths(datadir, 'xsessions')
 wayland_sessiondir = join_paths(datadir, 'wayland-sessions')
 schemadir = join_paths(datadir, 'glib-2.0', 'schemas')
-pkglibdir = join_paths(libdir, meson.project_name().to_lower())
+pkglibdir = libdir
 pkgdatadir = join_paths(datadir, meson.project_name().to_lower())
 po_dir = join_paths(meson.project_source_root(), 'po')
 
-- 
2.51.0

